src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -341,9 +341,9 @@ protected static function getBundleDefaultConfig()
             'secret' => 's3cr3t',
             'trusted_hosts' => [],
             'trusted_headers' => [
-                'x-forwarded-all',
-                '!x-forwarded-host',
```

Comment:
with a `!`, that's a test, why remove it?

Reply:
hmm, this is not a test, this is a expected default config.

There are no test on it.

Reply:
While adding a test this option, I wonder if this parameter shouldn't be deprecated (same for !x-forwarded-prefix) Without `-all` they doesn't make sens anymore.

Reply:
I wondered the same as @jderusse actually, might be worth deprecating :)

Reply:
deprecated